frontend/integration-tests/views/dashboard-shared.view.ts

```diff
@@ -0,0 +1,8 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { untilNoLoadersPresent } from '../views/crud.view';
```

Comment:
This import path will change to: `'@console/internal-integration-tests/views/crud.view'`

Reply:
Fixed